src/components/Login.js

```diff
@@ -0,0 +1,38 @@
+import { useForm } from "react-hook-form";
+import useAuth from "./hooks/useAuth";
+import { Link } from "react-router-dom";
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from "yup";
+import Navbar from "./Navbar";
+
+function Login(){
+
+    const schema = yup.object({
+        email: yup.string().email().required(),
+        password: yup.string().min(8).required()
+    })
+
+    const { login, loginError } = useAuth();
+    const { register, handleSubmit, watch, formState: { errors } } = useForm({
+        resolver: yupResolver(schema)
+    });
+
+    return(
+        <div className="App">
+            <Navbar>
+                <Link to="/register" style={{ float:"right" , marginRight:"10px"}}>Register</Link>
```

Comment:
e ja bih u login komponenti drzala samo tu formu i logiku vezanu za sam login - navbar je vec deo layouta - to bih stavila u App komponentu recimo